CSC8503CoreClasses/Room.h

```diff
@@ -14,6 +14,10 @@ namespace NCL {
 			Room(std::string roomPath);
 			~Room();
 			RoomType GetType() const { return mType; }
+			std::map<Vector3, TileType> GetTileMap() const { return mTileMap; }
+			std::vector<Light*> GetLights() const { return mLights; }
+			std::vector<Matrix4> GetCCTVTransforms() const { return mCCTVTransforms; }
+			std::vector<Vector3> GetItemPositions() const { return mItemPositions; }
```

Comment:
We're storing light pointers, but can we store the others as vectors of pointers, or similar? A second copy of matrices and vectors might get expensive. 
(I'm presuming there's a copy on the objects themselves?)